src/projector.cpp

```diff
@@ -20,6 +20,12 @@
 #include "src/projector.h"
 #include "src/jaz/gravis/t3Vector.h"
 #include <src/time.h>
+#include <cufft.h>
```

Comment:
Please keep the ability to compile RELION without CUDA SDK by guarding by `#ifdef CUDA`.

Reply:
fixed